File: Shared/Model/Artikal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Delos.Model
{
    public class artikal
    {
        [Key]
        public string sifra { get; set; }
        public string dobavljac_sifra { get; set; }
        public string dobavljac { get; set; }
        public string naziv { get; set; }
        public decimal kolicina { get; set; }
        public decimal cijena_sa_rabatom { get; set; }
        public decimal? cijena_prodajna { get; set; }
        public decimal? cijena_mp { get; set; }
        public string dostupnost { get; set; }
        public DateTime? zadnje_ucitavanje { get; set; }
        public List<string> slike { get; set; }
        public List<string> vrste { get; set; }
        public string kategorija { get; set; }
        public bool kalkulacija { get; set; }
        public string barkod { get; set; }
        public string brend { get; set; }
        public string garancija { get; set; }
        public string opis { get; set; }
        public bool aktivan { get; set; }
        public string vrsteString
        {
            get
            {
                var str = "";
                if (this.vrste != null)
                {
                    foreach (var s in this.vrste)
                        str += s + ";";
                }
                return str;
            }
        }
    }
}

File: Delos/ClientApp/src/app/model/artikal.ts
import { istorija_cijena } from "./artikal - Copy";

export class Artikal {
  sifra: string;
  naziv: string;
  dobavljac: string;
  dobavljac_sifra: string;
  kolicina?: number;
  dostupnost: string;
  cijena_sa_rabatom?: number;
  cijena_prodajna?: number;
  cijena_mp?: number;
  zadnje_ucitavanje: Date;
  slike?: string[];
  vrste?: string[];
  kategorija: string;
  opis: string;
  barkod: string;
  garancija: string;
  brend: string;
  aktivan: boolean;
  istorija_cijena: Array<istorija_cijena>;
}
